ethexe/service/src/tests.rs

```diff
@@ -1604,24 +1577,27 @@ mod utils {
         pub fn new_node(&mut self, config: NodeConfig) -> Node {
             let NodeConfig {
                 name,
-                sequencer_public_key,
-                validator_public_key,
-                validator_session_public_key,
-                network,
+                db,
+                validator_config,
                 rpc: service_rpc_config,
                 fast_sync,
             } = config;
 
-            let db = Database::from_one(&MemDb::default());
+            let db = db.unwrap_or_else(|| Database::from_one(&MemDb::default()));
+
+            let (network_address, network_bootstrap_address) = self
+                .bootstrap_network
+                .as_mut()
+                .map(|(_, bootstrap_address, nonce)| {
+                    *nonce += 1;
+
+                    if *nonce % MAX_NETWORK_PEERS_ONE_TEST == 0 {
```

Comment:
Actually, 1000 services in 1 test sound pretty impossible

Reply:
Just additional protection to save developers time in future. Somebody could change this const